core/src/main/antlr/apoc/custom/Signature.g4

```diff
@@ -16,7 +16,7 @@ opt_type:	base_type'?'? ;
 base_type:	'MAP' | 'ANY' | 'NODE' | 'REL' | 'RELATIONSHIP' | 'EDGE' | 'PATH' | 'NUMBER' | 'LONG' | 'INT' | 'INTEGER' | 'FLOAT' | 'DOUBLE' | 'BOOL' | 'BOOLEAN' | 'DATE' | 'TIME' | 'LOCALTIME' | 'DATETIME' | 'LOCALDATETIME' | 'DURATION' | 'POINT' | 'GEO' | 'GEOMETRY' | 'STRING' | 'TEXT' ;
 NEWLINE:	[\r\n]+ ;
 QUOTED_IDENTIFIER:	'`' [^`]+? '`' ;
-IDENTIFIER:	[a-zA-Z_][a-zA-Z0-9_]+ ;
+IDENTIFIER:	[a-zA-Z_][a-zA-Z0-9_]* ;
```

Comment:
fai revert di questo e dentro Singatures a linea 134 aggiungi questa linea:

`if (ns == null) throw new IllegalStateException("Unsupported procedure name, the procedure must have at least two chars ");`

e cambia i test di conseguenza